src/main/java/frc/robot/subsystems/ExampleSubsystem.java

```diff
@@ -7,9 +7,38 @@
 import edu.wpi.first.wpilibj2.command.CommandBase;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
+import edu.wpi.first.wpilibj.XboxController;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+
 public class ExampleSubsystem extends SubsystemBase {
+  
+  //motors
+  private final WPI_TalonSRX rightMotor;
+  private final WPI_TalonSRX rightFollower; 
+  private final WPI_TalonSRX leftMotor; 
+  private final WPI_TalonSRX leftFollower; 
+
+  //controller (more like CONTROLSLOLOL)
+  private final XboxController controller; 
+
   /** Creates a new ExampleSubsystem. */
-  public ExampleSubsystem() {}
+  public ExampleSubsystem() {
+    //right motors
+    rightMotor = new WPI_TalonSRX(0);
+    rightFollower = new WPI_TalonSRX(1);
+    rightFollower.follow(rightMotor);
+
+    //inverting
+    rightFollower.setInverted(true);
+    rightMotor.setInverted(true); 
+
+    //left motors
+    leftMotor = new WPI_TalonSRX(2);
+    leftFollower = new WPI_TalonSRX(3);
+    leftFollower.follow(leftMotor);
+
+    controller = new XboxController(0); //controllers
```

Comment:
The controller should be moved into robot container periodic, and the necessary joystick values should be passed into the subsystem through a function. This will allow us to put other robot functions on other buttons of the controller if needed.